src/site/_data/event.js

```diff
@@ -21,9 +21,13 @@
 const eventData = [
   {
     title: 'Day 1',
-    when: '2020-06-30T16:00Z',
-    duration: 8,
+    when: '2020-06-30T16:00Z', // 9am PST (-7)
```

Comment:
```suggestion
    when: '2020-06-30T16:00Z', // 9am PDT (-7)
```
nit, in case it's important.

PST is -8, PDT is -7.

We're in PDT now and we'll still be in PDT when the event happens. PDT ends Nov 1. I double checked the timestamp and 16:00 UTC == 09:00 am PDT 👍